### DISCUSSION TITLE
what is the difference between Backbone and unspecified network interface?

### DISCUSSION THREAD

**Discussion by Leo2442926161** [openthread/openthread#9383] [Q&A] [answered]
Hi all, I'd like to ask a simple question, what is the difference between Backbone and unspecified network interface?

From what I understand, the backbone interface is either Ethernet or wifi interface, so the messages are able to route to Internet. As for unspecified network interface, the messages will be route to Thread network, do I understanding correct?

I did a test for IP communication, I observed that thread device can ping IP devices directly via OTBR, why do we need to specify the netif then?  

Thanks very much in advance.

```
- netif: the network interface to bind to.
  - not specified: Thread network interface.
  - `-u`: unspecified network interface.
  - `-b`: Backbone network interface.
```

**Comment by jwhui** [comment, marked as answer]
Ability to specify the backbone interface is primarily used for testing. There are tests that need to send the UDP packet out the backbone interface, rather than the Thread interface.
